ci/tests/Jenkinsfile.e2e-prs

```diff
@@ -0,0 +1,72 @@
+pipeline {
+
+  agent { label 'linux1' }
+
+  parameters {
+    string(
+      name: 'BRANCH_NAME',
+      description: 'Name of the branch to checkout and build.',
+      defaultValue: 'develop',
+    )
+    string(
+      name: 'NETWORK',
+      description: 'Name of test network to use.',
+      defaultValue: 'ropsten',
+    )
+    string(
+      name: 'TEST_MARKERS',
+      description: 'Marker expression for matching tests to run.',
+      defaultValue: 'critical and high',
```

Comment:
We need "or" conjunction here (even I feel it may still work with and:) according to pytest doc: http://doc.pytest.org/en/latest/example/markers.html#reading-markers-which-were-set-from-multiple-places

Reply:
Good point.